browser/scss/kit/main.scss

```diff
@@ -6,6 +6,7 @@
 	img {
 		max-height: 200px
 	}
+
 }
 
 .kit {
```

Comment:
I actually deleted the span class here in my last commit... are you sure you're working off the latest master?